apps/src/makerlab/BoardController.js

```diff
@@ -4,6 +4,9 @@
 /* global Promise */
 
 try {
+  // Polyfill node's process.hrtime for the browser, gets used by johnny-five.
+  process.hrtime = require('browser-process-hrtime');
```

Comment:
From what I can tell, [webpack provides `process`](https://webpack.github.io/docs/api-in-modules.html#process) (though apparently doesn't provide the `.hrtime` utility that node's `process` does). This added line is in a try/catch, currently only targeting Chrome and only being used in active curriculum dev.